core/src/sigverifier/bls_sigverifier.rs

```diff
@@ -25,21 +39,224 @@ impl SigVerifier for BLSSigVerifier {
         packet_batches: Vec<PacketBatch>,
     ) -> Result<(), SigVerifyServiceError<Self::SendType>> {
         // TODO(wen): just a placeholder without any batching.
-        if let Some(sender) = &self.sender {
-            packet_batches.iter().for_each(|batch| {
-                batch.iter().for_each(|packet| {
-                    if let Ok(message) = packet.deserialize_slice::<BLSMessage, _>(..) {
-                        sender.send(message).unwrap();
+        let mut packet_received = 0;
+        let mut bls_messages_sent = 0;
+        let mut bls_messages_malformed = 0;
+        let mut sent_failed = 0;
```

Comment:
if we go the accumulate route, could replace these with a local `BLSSigVerifierStats`

Reply:
Done.